httpobsdashboard/templates/index.html

```diff
@@ -104,17 +104,17 @@
                         </span>
                     </td>
                     <td class="text-center">
-                        <span class="grade" data-content="{{ results.httpobs.delta90 }}" data-pass="{% if results.httpobs.delta90 > 0 %}True{% elif results.httpobs.delta90 == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
+                        <span class="grade" data-content="Over the past 90 days there has been a score change of: <strong>{{ results.httpobs.delta90 }}/100</strong>" data-pass="{% if results.httpobs.delta90 > 0 %}True{% elif results.httpobs.delta90 == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
                         {{ results['httpobs']['delta90'] }}
                         </span>
                     </td>
                     <td class="text-center">
-                        <span class="grade" data-content="{{ results.httpobs.delta }}" data-pass="{% if results.httpobs.delta > 0 %}True{% elif results.httpobs.delta == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
+                        <span class="grade" data-content="Since records began there has been a score change of: <strong>{{ results.httpobs.delta }}/100</strong>" data-pass="{% if results.httpobs.delta > 0 %}True{% elif results.httpobs.delta == 0 %}None{% else %}False{% endif %}" data-toggle="popover">
```

Comment:
Currently the top score is 135:

https://github.com/mozilla/http-observatory/blob/master/httpobs/docs/scoring.md